plugins/usi-dock/fu-usi-dock-mcu-device.c

```diff
@@ -563,23 +594,15 @@ fu_usi_dock_mcu_device_write_firmware_with_idx(FuUsiDockMcuDevice *self,
 	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 42, "internal-flash");
```

Comment:
also, there's still FU_PROGRESS_FLAG_GUESSED here -- doesn't the daemon spit out some big red text on what the values are supposed to be? See https://github.com/fwupd/fwupd/wiki/Daemon-Warning:-FuProgress-steps